geniza/people/models.py

```diff
@@ -0,0 +1,27 @@
+from django.db import models
+from django.utils.translation import gettext_lazy as _
+
+
+class Person(models.Model):
+    """Person mentioned in or related to a geniza document."""
+    name = models.CharField(max_length=100)
+    profession = models.ForeignKey("Profession", null=True, on_delete=models.CASCADE)
```

Comment:
This means that if I delete the "Scribe" profession, that deletes all Persons who are scribes, correct? Is that the functionality we want?

Reply:
Probably not, I imagine we'd do `models.SET_NULL` in the real version!